backend/compact-connect/tests/unit/test_app.py

```diff
@@ -187,6 +190,29 @@ def test_synth_local_ui_port_override(self):
         )
         self._inspect_api_stack(app.sandbox_stage.api_stack)
 
+    def test_synth_generates_provider_user_pool_with_expected_custom_attributes(self):
+        """
+        Test infrastructure as deployed in a developer's sandbox
+        """
+        context = self._when_testing_sandbox_stack_context()
+
+        app = CompactConnectApp(context=context)
+        persistant_stack = app.sandbox_stage.persistent_stack
```

Comment:
nit: `persistent_stack`? :-)

Reply:
ah, took me a minute to see what the issue was. I'll get that fixed

Reply:
Ha, sorry! Should have been clearer.